proxyUtils.js

```diff
@@ -0,0 +1,24 @@
+const envVariables = require('./config/environment');
+
+const decoratePrivateRequestHeaders = function () {
+  return function (proxyReqOpts, srcReq) {
+    proxyReqOpts.headers["Content-Type"] = "application/json";
```

Comment:
Why dont you call decoratePublicRequestHeaders & then append specific to headrs like userToken & channelId

Reply:
@vinukumar-vs resolved.